client/components/select-dropdown/index.jsx

```diff
@@ -207,12 +207,20 @@ class SelectDropdown extends Component {
 			} );
 		}
 
-		let dropdownClassName = classNames( dropdownClasses );
-		let selectedText = this.props.selectedText
-			? this.props.selectedText
-			: result( find(
-				this.props.options, { value: this.state.selected }
+		const dropdownClassName = classNames( dropdownClasses );
+
+		let selectedText = this.props.selectedText;
```

Comment:
Almost, 😄 I think 68fbbcb snuck in. Perhaps make sure your local master is up to date.